backend/hardhat.config.ts

```diff
@@ -48,6 +54,26 @@ async function setReward(hre: typeof import('hardhat'), quizContract: any, rewar
   console.log(`Set reward to ${reward} ROSE. Transaction hash: ${receipt!.hash}`);
 }
 
+async function addAllowMint(hre: typeof import('hardhat'), contract: any, address: string) {
+  const tx = await contract.addAllowMint(address);
+  await tx.wait();
+  console.log(`Address ${address} allowed to mint.`);
+}
+
+async function setNftAddress(hre: typeof import('hardhat'), contract: any, nftAddress: string) {
+  const tx = await contract.setNftAddress(nftAddress);
+  await tx.wait();
+  console.log(`NFT address set to ${nftAddress}.`);
+}
+
+async function setCustomNFT(hre: typeof import('hardhat'), contract: any, jsonMetadata: string) {
+  console.log(`Setting custom NFT with metadata: ${jsonMetadata}`);
+  console.log(`NFT address: ${await contract.getNftAddress()}`);
+  const tx = await contract.setCustomNFT(jsonMetadata, { gasLimit: 15_000_000 });
+  await tx.wait();
+  console.log(`Custom NFT set.`);
+}
+
```

Comment:
Move this and all other helpers not directly called in the tasks into src/ folder. Let's make hardhat.config.ts slimmer.

Reply:
Moved to src/ folder.